src/openrct2/actions/StaffHireNewAction.hpp

```diff
@@ -204,7 +204,8 @@ DEFINE_GAME_ACTION(StaffHireNewAction, GAME_COMMAND_HIRE_NEW_STAFF_MEMBER, Staff
             PeepSpriteType spriteType = spriteTypes[_staffType];
             if (_staffType == static_cast<uint8_t>(StaffType::Entertainer))
             {
-                spriteType = static_cast<PeepSpriteType>(PEEP_SPRITE_TYPE_ENTERTAINER_PANDA + _entertainerType);
+                spriteType = static_cast<PeepSpriteType>(
+                    PEEP_SPRITE_TYPE_ENTERTAINER_PANDA + static_cast<uint8_t>(_entertainerType));
```

Comment:
This code seems messy. It'd be better to introduce an array or function that maps a type to a sprite.

Reply:
I agree, it's a bit odd to do this sort of arithmetic, and I don't like how there's multiple casts in the one line.

I'll try writing a function that takes in one EntertainerCostume and returns a PeepSpriteType.

Reply:
In peep/Staff.cpp there's already a function 
`void Staff::SetCostume(uint8_t value)
{
    SpriteType = static_cast<PeepSpriteType>(value + PEEP_SPRITE_TYPE_ENTERTAINER_PANDA);
}`

Should I make another function there,
`PeepSpriteType Staff:GetEntertainerCostumeSprite(EntertainerCostume entertainerCostume)
{
   ...
   return entertainerCostumeSprite
}` ?
Or maybe there's a better file it to put it in? I'm new to the project and actually not that experienced with modifying large C++ projects.